openshift/templates/images/nodejs/s2i/api/run

```diff
@@ -8,9 +8,6 @@
 #
 
 echo "Running migration $(npx knex migrate:latest)" >&2
-echo "seed:run 01_capabilities.js $(npx knex seed:run --specific '01_capabilities.js')" >&2
-echo "seed:run 02_role_capabilities.js $(npx knex seed:run --specific '02_role_capabilities.js')" >&2
 echo "seed:run 08_picker_options.js $(npx knex seed:run --specific '08_picker_options.js')" >&2
```

Comment:
how come when I run production seeds locally, I use a different command?

shouldn't this command be
`npx knex seed:run --specific=999_production_seeder.js` ?

Reply:
and if not, 

why?

and what should the command be for a local seeding?

and what should the command be changed to in the README for the pmo-mssql-converter?

Reply:
not running production seeds, we are just running picker options, which is the only seed that we should be running.
Also not sure how it is different still using `npx knex seed:run` 
also don't understand your ref to pmo-mssql-converter, maybe reach out in chat for this one